wire-ios-request-strategy/Sources/Notifications/Push Notifications/Notification Types/Content/ZMLocalNotification+Events.swift

```diff
@@ -191,6 +192,17 @@ private class ConversationCreateEventNotificationBuilder: EventNotificationBuild
         LocalNotificationType.event(.conversationCreated)
     }
 
+    override func shouldCreateNotification() -> Bool {
```

Comment:
For now I only provided fixes for `ConversationCreateEventNotificationBuilder` and `NewMessageNotificationBuilder`, since the ticket was explicitly mentioning these two types.
Maybe it makes sense to extend the workaround.